biggraphite/accessor.py

```diff
@@ -870,6 +870,11 @@ def flush(self):
         """Flush any internal buffers."""
         pass
 
+    @abc.abstractmethod
+    def clear(self):
+        """Clear all internal data."""
+        pass
```

Comment:
This method seems confusing: 

* `flush` clears internal buffers
* `clean` removes outdated metrics
* `drop_all_metrics` drops all data and metadata

What is this one supposed to to compared to the other ones?

Reply:
It sure is confusing. Let's discuss it outside this PR, right ?

Reply:
Sure. If you want to merge this PR before we talk about it, could you provide a more specific description of what the method should do (and what you intend it to do)?